app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -57,7 +59,6 @@
 
     FirebaseAuth fAuth;
     private static final String TAG = "ActDescActivity";
-    FirestoreActivityManager firestoreManager;
 
     DrawerLayout drawerLayout;
```

Comment:
A lot of these fields can be removed if you use the new `Navigation` class

Reply:
Yes, you're completely right, those will be removed as I will merge my branch with the main, that's good news for our code coverage!